WORKSPACE

```diff
@@ -447,15 +447,13 @@ http_archive(
         "//:patches/v8/0010-Enable-V8-shared-linkage.patch",
         "//:patches/v8/0011-Randomize-the-initial-ExecutionContextId-used-by-the.patch",
         "//:patches/v8/0012-Always-enable-continuation-preserved-data-in-the-bui.patch",
-        "//:patches/v8/0013-Attach-continuation-context-to-Promise-thenable-task.patch",
-        "//:patches/v8/0014-increase-visibility-of-virtual-method.patch",
-        "//:patches/v8/0015-Add-ValueSerializer-SetTreatFunctionsAsHostObjects.patch",
-        "//:patches/v8/0016-wasm-liftoff-arm64-Fix-LoadTaggedPointer.patch",
+        "//:patches/v8/0013-increase-visibility-of-virtual-method.patch",
```

Comment:
I assume the continuation context patch is removed because 12.4 has the upstream fix available?

Reply:
Yep! this test was already removed in upstream